packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1707,8 +1718,13 @@ export default mixins(
 				// it visibly stays behind free floating without a connection.
 				connection.removeOverlays();
 
+				const sourceEndpoint = connection.endpoints && connection.endpoints.length && connection.endpoints[0];
```

Comment:
isn't this sufficient? Empty array would also mean its undefined/false
```suggestion
				const sourceEndpoint = connection.endpoints && connection.endpoints[0];
```

Reply:
ya I guess it's the same

Reply:
updated